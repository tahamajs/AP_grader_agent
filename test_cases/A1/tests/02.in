add_task NonExistentProject Task1
list_tasks NonExistentProject
list_projects
