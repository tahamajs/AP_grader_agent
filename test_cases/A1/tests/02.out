Project not found: NonExistentProject
Project not found: NonExistentProject
Projects:
